src/rules/function-url-no-scheme-relative/README.md

```diff
@@ -0,0 +1,45 @@
+# function-url-no-scheme-relative
+
+Disallow scheme-relative urls.
+
+```css
+a { background-image: url('//www.google.com/file.jpg'); }
+/**                        ↑ 
+ *  This scheme-relative url */
+```
+
+A scheme-relative url is a url that begins with // followed by a host.
```

Comment:
"A [scheme-relative url](https://url.spec.whatwg.org/#syntax-url-scheme-relative) is a url that begins with `//` followed by a host."